ltp-production.yaml

```diff
@@ -1758,3 +1758,15 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5953
     active: true
     intermittent: true
+  - environments: *environments_all
```

Comment:
if its only failing with compat mode, why disable for all environments?

Reply:
same as above reason.